### DISCUSSION TITLE
Phishing site report - Followup to comment #101505

### DISCUSSION THREAD

**Discussion by sdpollack** [community/community#101664] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I'm trying again to report a phishing site hosted on Github Pages.  The site is https://pay-pal-customser-service.github.io/care/.

I submitted comment #101505 yesterday and was told to visit https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam.  None of the options listed on that page work in my situation.  

I don't know who the user is. There is no issue or pull request to report from. This is not from a discussion or comment. I looked for the app in Marketplace but it was not listed. It is not a contact link.

I see no way to report abuse in this situation. Clicking the various Contact Support links do not lead to a way to report the issue, they only lead back to knowledge bases or the community site. Seriously, you've locked down the process to submit tickets so tightly that there is no way for me to report this.

Please advise.  

My feedback is that there needs to be an easier and clearer way to report abuse in this situation.  Github has hidden this too well.

And **please** take down this site immediately.  My mother was almost scammed by it.

**Comment by yoannchaudet** [comment]
Thank you for your report.

For this kind of site, we are aware the process is not simple enough. The repository this site is associated with is: https://github.com/pay-pal-customser-service/care. And there is a report button on that page.

I went ahead and forwarded your message to our trust and safety team internally so they can review and take an action asap.
